.cspell.json

```diff
@@ -17,6 +18,7 @@
     "**/*.pcd",
     "**/*.pt",
     "**/*.stl",
+    "**/*.svg",
```

Comment:
We don't ignore SVG files, instead, add some patterns to ignore.
https://github.com/tier4/autoware-spell-check-dict/blob/29c6362fdbd20e379db1763fc6168c4744558f86/.cspell.json#L153-L164

```suggestion
```

Reply:
Got it. I'll add some RegExp to pass it.